src/components/ColorCombos/index.js

```diff
@@ -90,6 +92,8 @@ export default class ColorCombos extends Component {
                               fg.originalIndex,
                               getAllColorTypes(suggestion.fg)
                             );
+
+                            this.listItem.focus();
```

Comment:
Should we guard this with:

```js
if (this.listItem) {
  this.listItem.focus()
}
```

Reply:
🤷‍♂️ I don't feel too strongly about this and I could go either way.  I will add the guard